OMEdit/OMEditLIB/Plotting/PlotWindowContainer.h

```diff
@@ -65,9 +65,12 @@ class PlotWindowContainer : public QMdiArea
   bool isPlotWindow(QObject *pObject);
   bool isAnimationWindow(QObject *pObject);
   bool isDiagramWindow(QObject *pObject);
+  bool isUniqueName(QString name);
   bool eventFilter(QObject *pObject, QEvent *pEvent);
 private:
+  void addRenameTabToSubWindowSystemMenu(QMdiSubWindow *pMdiSubWindow);
   DiagramWindow *mpDiagramWindow;
+  QAction *mpRenamePlotWindowAction;
```

Comment:
Remove this as well.

Reply:
@adeas31 Good Catch will do!
Should I make an issue as well s.t this appears in the patch note when Francesco writes the release?

Reply:
Yeah sure.